external/curl/Makefile

```diff
@@ -0,0 +1,85 @@
+###########################################################################
+#
+# Copyright 2018 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+# curl/Makefile
+
+-include $(TOPDIR)/.config
+-include $(TOPDIR)/Make.defs
+
+DEPPATH	= --dep-path .
+VPATH		=
+
+ASRCS =
+
+# CURL common & feature specific source files
```

Comment:
If possible, You may have to Segregate these huge set of files based on common and feature specific.
It would have helped to understand the dependencies of files per feature.

Reply:
We will optimize curl in consecutive patches. In first release we are targeting basic functionality.